example/project.clj

```diff
@@ -1,19 +1,22 @@
 (defproject lein-monolith.example/all "MONOLITH"
   :description "Overarching example project."
 
+  :pedantic? :ranges
```

Comment:
Unit tests seemed to throw a `:abort and ranges have a type mismatch merging profiles` error before this. From what I could tell `:ranges` should already be default here but I could just be misunderstanding the real problem here 🤔

Reply:
Huh... this definitely shouldn't be required.

Reply:
Seeing the same thing, should be defaulting to [:ranges](https://github.com/technomancy/leiningen/blob/64e02a842e7bb50edc9b8b35de1e2ef1fac090dd/sample.project.clj#L82)  🤷